src/main.rs

```diff
@@ -63,19 +71,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Comments
     println!("\n-------- COMMENTS \u{1F4AC} --------\n");
     let comments = &data["Comment"]["Comments"]["CommentsList"];
-    let comments_len = comments
-        .as_array()
-        .map(|array| array.len())
-        .unwrap_or(0);
+    let comments_len = comments.as_array().map(|array| array.len()).unwrap_or(0);
     println!("You published {comments_len} comments\n");
 
     // Videos & likes
     println!("\n-------- YOUR ACCOUNT'S STATS \u{1F464} --------\n");
     let videos = &data["Video"]["Videos"]["VideoList"];
-    let videos_len = videos
-        .as_array()
-        .map(|array| array.len())
-        .unwrap_or(0);
+    let videos_len = videos.as_array().map(|array| array.len()).unwrap_or(0);
```

Comment:
```suggestion
    let videos_len = videos.as_array().map(|array| array.len()).unwrap_or(0);
```
Same as on line 74 : I think that the other syntax is more readable